pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/JuniperKitCreationTest.java

```diff
@@ -1,43 +1,66 @@
 package org.broadinstitute.dsm.juniperkits;
 
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
 import java.util.List;
 import java.util.Random;
 
 import com.google.gson.Gson;
-import com.typesafe.config.Config;
-import org.broadinstitute.dsm.DSMServer;
-import org.broadinstitute.dsm.TestHelper;
+import org.broadinstitute.dsm.DbTxnBaseTest;
+import org.broadinstitute.dsm.db.DDPInstance;
 import org.broadinstitute.dsm.db.KitRequestShipping;
 import org.broadinstitute.dsm.model.nonpepperkit.JuniperKitRequest;
 import org.broadinstitute.dsm.model.nonpepperkit.KitResponse;
+import org.broadinstitute.dsm.model.nonpepperkit.KitResponseError;
 import org.broadinstitute.dsm.model.nonpepperkit.NonPepperKitCreationService;
-import org.broadinstitute.dsm.statics.ApplicationConfigConstants;
+import org.broadinstitute.dsm.util.EasyPostUtil;
+import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+
+/**
+ * Tests the NonPepperKitCreationService class
+ * To run this class use the following VM variables
+ * -ea -Dconfig.file=[path to /pepper-apis/output-build-config/testing-inmemorydb.conf]
+ *  */
+
+@RunWith(MockitoJUnitRunner.class)
+public class JuniperKitCreationTest extends DbTxnBaseTest {
 
-public class JuniperKitCreationTest {
     NonPepperKitCreationService nonPepperKitCreationService = new NonPepperKitCreationService();
-    private Config cfg;
+    final String instanceGuid = "Juniper-mock-guid";
+    final String instanceName = "Juniper-mock";
+    final String salivaKitType = "SALIVA";
+    final String bspPrefix = "JuniperTestProject";
+    DDPInstance ddpInstance;
+    EasyPostUtil mockEasyPostUtil = mock(EasyPostUtil.class);
+
+
 
     @Before
-    public void beforeClass() {
-        TestHelper.setupDB();
-        cfg = TestHelper.cfg;
+    public void setupJuniperBefore() {
+        JuniperSetupUtil.setupUpAJuniperInstance(instanceName, instanceGuid, "Juniper-Mock", bspPrefix);
+        JuniperSetupUtil.loadDSMConfig();
+        ddpInstance = DDPInstance.getDDPInstanceWithRoleByStudyGuid(instanceGuid, "juniper_study");
+    }
 
-        DSMServer.setupDDPConfigurationLookup(cfg.getString(ApplicationConfigConstants.DDP));
+    @After
+    public void deleteJuniperInstance() {
+        JuniperSetupUtil.deleteJuniperTestStudies();
     }
 
     @Test
-    @Ignore
     public void createNewMockJuniperKitTest() {
```

Comment:
Typically test method names include the primary thing they are testing, like myMethodTest. That convention makes it easy for readers to understand the intent of the test without Java doc.
In this case is that createNonPepperKit? The term Mock does not make sense here unless we are testing a mock, per se.